src/Algebra/Morphism/Structures.agda

```diff
@@ -694,6 +694,48 @@ module LoopMorphisms (L₁ : RawLoop a ℓ₁) (L₂ : RawLoop b ℓ₂) where
 
     open IsLoopMonomorphism isLoopMonomorphism public
 
+------------------------------------------------------------------------
+-- Morphisms over Kleene algebra structures
+------------------------------------------------------------------------
+module KleeneAlgebraMorphisms (R₁ : RawKleeneAlgebra a ℓ₁) (R₂ : RawKleeneAlgebra b ℓ₂) where
+
+  open RawKleeneAlgebra R₁ renaming
+    ( Carrier to A; _≈_ to _≈₁_
+    ; _⋆ to _⋆₁
+    ; rawSemiring to rawSemiring₁
+    )
+
+  open RawKleeneAlgebra R₂ renaming
+    ( Carrier to B; _≈_ to _≈₂_
+    ; _⋆ to _⋆₂
+    ; rawSemiring to rawSemiring₂
+    )
+
+  open MorphismDefinitions A B _≈₂_
+  open FunctionDefinitions _≈₁_ _≈₂_
```

Comment:
Test fails here: imported module `Function.Definitions` as `FunctionDefinitions` is not (no longer) parametrised on the underlying setoid relations, this `open` fails.

Reply:
Actually this failure is pervasive from the top of the module... evidence of not keeping pace with the refactoring of `Function` ...